mautic/mautic
# [UI] Remove minimize sidebar feature

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR removes the minimize sidebar feature, based on [community votes in Slack](https://mautic.slack.com/archives/C024XBCQS3Z/p1721932963539929).
It also considers user feedback regarding hiding things that just don't work fully.

Until now, we had an issue with the sidebar, where in the minimized state, everything under a submenu becomes inaccessible. So replacing we can focus on improving the experience in different ways.

This PR also removes old code related to the right sidebar, replaced recently.






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Dashboard and check if you can use the sidebar properly.
3. Using your browser's developer tools, activate the mobile mode and check if the sidebar can be accessible and keeps working normally.
